src/haxeLanguageServer/features/haxe/RenameFeature.hx

```diff
@@ -116,6 +116,7 @@ class RenameFeature {
 			srcFolders = context.config.user.renameSourceFolders;
 		}
 
+		Sys.setCwd(workspacePath);
```

Comment:
I don't think we should go around and change current working directory in a some feature's handler.
maybe we should `Path.join` `workspacePath` with each entry in `srcFolders` so that it runs on absolute paths. but I'm not sure if that messes up file lookup.

Reply:
I reworked the RenameFeature. It now works with absolute paths and does not require the CWD to be set.